content/ember/v3/deprecate-window-global.md

```diff
@@ -0,0 +1,40 @@
+---
+id: ember-source.window-global
+title: Ember Global
+until: 4.0.0
+since: 3.26.0-beta.1
```

Comment:
```suggestion
since: 'Upcoming Features'
```
Until the feature is actually released in a stable Ember we actually release deprecations under the "Upcoming Features" version (I know the way to do this is a bit janky but it works 😂 ) 

We could get this merged if you want as soon as this change is applied